assets/js/lib/test-utils/factories/executions.js

```diff
@@ -83,3 +113,70 @@ export const withCompletedResults = (execution, result, checkResults) => ({
 export const checksExecutionCompletedFactory = Factory.define(({ params }) =>
   withCompletedResults(checksExecutionRunningFactory.build(params))
 );
+
+export const withEmptyExpectations = (checkResult) => {
+  const agents = checkResult.agents_check_results.map((agent) => ({
+    ...agent,
+    expectation_evaluations: [],
+  }));
+
+  return {
+    ...checkResult,
+    agents_check_results: agents,
+    expectation_results: [],
+  };
+};
+
+const addExpectation = (checkResult, name, expec, result) => {
+  const { type } = expec;
```

Comment:
expect maybe?

Reply:
This should be `expectation` to me, you're right